soft_constraint:
    lower:
        - -570.0
        - -1.97
        - -2.68
        - -2.12
        - 0.0
        - 0.0
        - 0.0
        - 0.0
        - 0.0
    upper:
        - 570.0
        - 1.97
        - 2.68
        - 2.12
        - 0.0
        - 0.0
        - 0.0
        - 0.0
        - 0.0

#END
